src/mcts/search.cc

```diff
@@ -70,6 +70,63 @@ MoveList MakeRootMoveFilter(const MoveList& searchmoves,
   return root_moves;
 }
 
+class MEvaluator {
```

Comment:
I assume we're confident that the optimizer will optimize this away entirely....

Reply:
I'll follow up on this.